docs/source/contributing/contributing.md

```diff
@@ -2,15 +2,15 @@
 
```

Comment:
This is a move?  Doesn't README.md link to this?

Reply:
Ah, I missed that. Sorry -- yes I will need to adjust the README.md link. 

I agree, I was lazy and just included the sphinx makefile. I'll teach myself the syntax and add that. I imagine I can port the logic present in `subdir.mk` for notebooks?

Reply:
> I agree, I was lazy and just included the sphinx makefile. I'll teach myself the syntax and add that. I imagine I can port the logic present in subdir.mk for notebooks?

You could do that or you could call out using recursive make.  Given that this is a standardized makefile, I would do the latter.

Reply:
> You could do that or you could call out using recursive make. Given that this is a standardized makefile, I would do the latter.

https://github.com/spacetx/starfish/issues/536